fvm/evm/types/errors.go

```diff
@@ -80,7 +108,7 @@ func (err DatabaseError) Error() string {
 	return fmt.Sprintf("database error: %v", err.err)
 }
 
-// IsADatabaseError returns true if the error or any underlying errors
+// IsDatabaseError returns true if the error or any underlying errors
```

Comment:
Did you want to rename the function as well?

Reply:
fixed it.